src/security/pausable.cairo

```diff
@@ -19,20 +19,20 @@ mod PausableComponent {
     }
 
     #[event]
-    #[derive(Drop, starknet::Event)]
+    #[derive(Drop, PartialEq, starknet::Event)]
```

Comment:
does this have any effect in the low level representation or is it fully compatible with Events from previous versions?

Reply:
I did ask Ariel about it when he suggested the change, and It is compatible, and shouldn't even modify the contract source code (class hash) since an optimization in the compiler ignores it if is not used, and it is only used in tests.